packages/@okta/vuepress-site/docs/reference/rate-limits/index.md

```diff
@@ -31,6 +31,25 @@ If any org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
 > * Rate limits may be changed to protect customers. We provide advance warning of changes when possible.
 > * You can expand the Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
+
+## Burst rate limits
+
+Okta provides rate limits for orgs based on the traffic that they expect to have. If your org experiences higher traffic than what is expected, this unplanned usage may potentially have an impact on end users. To ensure our customers are successful and to help minimize impact, Okta will not suspend usage above the rate limit (specifically for authentication and authorization flows). In the event that there is sustained usage above the purchased rate limit, Okta will require that you purchase an applicable offering that is in line with your usage. By making burst rate limits available, Okta provides peace of mind by ensuring an unplanned spike doesn't detrimentally affect the end user's experience.
```

Comment:
```suggestion
Okta provides rate limits for orgs based on the traffic they expect to have. If your org experiences higher traffic than expected, this unplanned usage may potentially have an impact on end users. To ensure our customers are successful and to help minimize impact, Okta will not suspend usage above the rate limit (specifically for authentication and authorization flows). In the event of sustained usage above the purchased rate limit, Okta will require you purchase an applicable offering in line with your usage. By making burst rate limits available, Okta provides peace of mind by ensuring an unplanned spike doesn't detrimentally affect the end user's experience.
```

Reply:
@hmobius, this paragraph was written the way that it was due to:
1. Legal reasons and legal reviewers
2. Best practice for localization